src/archivematica/dashboard/settings/components/oidc_auth.py

```diff
@@ -51,6 +51,11 @@
 
 OIDC_RP_SIGN_ALGO = os.environ.get("OIDC_RP_SIGN_ALGO", "HS256")
 
+OIDC_USE_PKCE = os.environ.get("OIDC_USE_PKCE", "false")
```

Comment:
Should this setting be converted into a boolean value? As discussed in https://github.com/artefactual/archivematica-storage-service/pull/764 it is initially extracted as a string and will be considered truth-y by `mozilla-django-oidc`.

This issue can be demonstrated by setting `OIDC_USE_PKCE` to `"false"` in the `tests/integration/docker-compose.yml` file and then running the integration OIDC tests. Since the realm configuration in the test mandates the use of PKCE, the existing OIDC tests are expected to fail when PKCE is disabled. However, the tests still pass, indicating the PKCE was used anyway.

Reply:
I have added a commit that should address this.